src/app/summary/summary.service.ts

```diff
@@ -0,0 +1,78 @@
+import {Injectable, OnDestroy} from '@angular/core';
+import {map} from "rxjs/operators";
+import {Subject} from "rxjs";
+import {ProjectsService} from "../projects/services/projects.service";
+
+export interface ProjectCount {
+  group: string;
+  count: number;
+  cellCount: number;
+}
+
+@Injectable({
+              providedIn: 'root'
```

Comment:
I think this would be better as not a singleton. It is only used in one page and is not needed for the whole duration of the user session, just while they are viewing the summary page

Please refer to the implementation of ProjectsService for an example

Reply:
Will do.

Reply:
Is this urgent for this release? @jacobwindsor

Reply:
@amnonkhen it's a very quick change and will improve performance